src/components/dashboard/Dashboard.tsx

```diff
@@ -136,13 +135,11 @@ function DashboardContent() {
                         {mainListItems}
                     </List>
                 </Drawer>
+                {/**Swapped out the use of theme for background color for white background */}
```

Comment:
Why?

Reply:
Because the background is white. I have removed the comment